adam-casey-devetry/showmechange-FE
# Michael state capitals

What I did:

- Made two components (ListOfStates)(ListOfCapitals)

- Inside each component I'm looping over a list of states and list of capitals displaying them on the screen separating them based on their color

- Inside of App.js I'm importing the new components 

To Test:

- Start the app

- Visit the home page 

  -verify that (ListOfStates) and (ListOfCapitals) are displaying 

  -Make sure states and capitals are the correct font color
